test/verify/check-system-info

```diff
@@ -193,6 +193,12 @@ class TestSystemInfo(MachineCase):
         b.wait_in_text('#system_information_hostname_text', "Adventure Box (host1.cockpit.lan)")
         self.assertEqual(m.execute("hostname").strip(), "host1.cockpit.lan")
 
+        transient_hostname = m.image.split('-')[0]
```

Comment:
Let's not make this assumption. As the tests show, at least on Debian and RHEL 8 the host name ends up as "localhost". I suggest to call `hostnamectl --transient` to detect the current transient host name. However, this would still assume that the transient host name is actually different from the static one, which may not be the case on all images. Hence, let's drop this altogether. See below.